pubsub/topics/create_topic_kinesis_ingestion.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package topics
+
+// [START pubsub_create_topic_with_kinesis_ingestion]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/pubsub"
+)
+
+func createTopicWithKinesisIngestion(w io.Writer, projectID, topicID, streamARN, consumerARN, awsRoleARN, gcpSA string) error {
+	// projectID := "my-project-id"
+	// topicID := "my-topic"
+	// streamARN := "stream-arn"
```

Comment:
It looks like from the test that all of these are static values; if so create them as local variables rather than args. https://googlecloudplatform.github.io/samples-style-guide/#minimal-arguments

Reply:
I initialized the values in the sample itself, is this what you were suggesting?